scalyr_agent/agent_main.py

```diff
@@ -413,7 +415,7 @@ def __start(self, quiet, no_fork, no_check_remote):
             self.__fail_if_already_running()
 
             if not quiet:
-                print "Configuration and server connection verified, starting agent in background."
+                print "Starting agent in background."
```

Comment:
We would like to emit as few lines as possible -- users tend to not read the output so it's better in our opinion to put as much information as we can in each line.

How about if you just rewrite this as:

    if not quiet:
        if no_check_remote:
            print "Configuration verified, starting agent in background."
        else
            print "Configuration and server connection verified, starting agent in background."